### DISCUSSION TITLE
Python 2 support

### DISCUSSION THREAD

**Discussion by Chris2011** [albilu/netbeansPython#23] [Ideas]
Afaik, Python 3 is not a replace of Python 2 or maybe not in the next couple of years. So is there already a plan to also support Python 2? Maybe with a switch inside the options for the project etc. I dunno how the status of python 2 is to be honest but afaik it is still available. Maybe just in legacy projects nowadays.

**Comment by albilu** [comment]
Python 2 is not supported anymore and it is recommended to move up.
Anyway it is supported by this plugin. 
The requirement of minimal Python 3.7 is for the LSP environment. Once installed you can add a Python 2.7 interpreter via the Platform management and be able to work projects with it.

Indeed there was a small bug preventing the Python 2.7 interpreter to be added which is fixed now in the latest release.
